pkg/config/app/kuma-dp/config.go

```diff
@@ -47,6 +47,10 @@ var DefaultConfig = func() Config {
 			PrometheusPort:            19153,
 			CoreDNSLogging:            false,
 		},
+		VirtualProbesServer: VirtualProbesServer{
+			Enabled: false,
+			Port:    9000,
+		},
```

Comment:
What was the motivation for not having this in the `dataplane.yaml` ?

Reply:
"Virtual Probes" is an existing concepts and it was configured using either a pod annotation or the kuma dp injector configuration with env `KUMA_RUNTIME_KUBERNETES_VIRTUAL_PROBES_PORT`.
Previously, we captured pod probe info on `Dataplane` objects because probe paths can be different on multiple pods. The new way does not care about probe paths defined by users and all user defined information is supported nearly "as-is". So we don't need this information from the `dataplane.yaml` any more.

Reply:
In addition, as you suggested, we should keep the capability of falling back to the old way. So I'm going to keep the path information on pods in new commits.